shim/linux/src/entry.c

```diff
@@ -66,6 +86,28 @@ dispatch_system_kvm_check_extension(void)
 static long
 dispatch_system_kvm_create_vm(void)
 {
+    char vmname[22];
+    struct shim_vm_t *vm =
+        (struct shim_vm_t *)vmalloc(sizeof(struct shim_vm_t));
+    if (NULL == vm) {
+        goto vm_free;
+    }
+    if (handle_system_kvm_create_vm(&vm->vmid)) {
+        goto vm_free;
+    }
+    snprintf(vmname, sizeof(vmname), "kvm-vm:%d", vm->vmid);
+    int32_t fd = anon_inode_getfd(vmname, &fops_vm, vm, O_RDWR | O_CLOEXEC);
+    if (fd < 0) {
```

Comment:
0 is a magic number. You need to use a predefined macro for this.

Reply:
#define MV_SHIM_INVALID_ID ((uint16_t)0x0) in mv_constants.h ....just want to double check